Documentation/ceph-object.md

```diff
@@ -57,7 +63,59 @@ kubectl create -f object.yaml
 kubectl -n rook-ceph get pod -l app=rook-ceph-rgw
 ```
 
+### Connect to external Ceph Gateways
+
+Rook can connect to existing RGW gateways, this works in conjunction with the external mode of the `CephCluster` CRD.
+So if you have an external `CephCluster` CR, you can instruct Rook to consume external gateways with the following:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: my-store
+  namespace: rook-ceph
+spec:
+  gateway:
+    # the port on which **ALL** the gateway(s) are listening on
+    # passing a single IP from a load-balancer is valid too
+    port: 8080
+    externalRgwEndpoints:
+      - ip: 192.168.39.182
+```
+
+You can use the existing `object-external.yaml` file.
+When ready the ceph-object-controller will output a message in the Operator log similar to this one:
+
+```log
+ceph-object-controller: ceph object store gateway service running at 10.100.28.138:8080
+```
+
+You can now get and access the store via:
+
+```console
+kubectl -n rook-ceph get svc -l app=rook-ceph-rgw
+NAME                     TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+rook-ceph-rgw-my-store   ClusterIP   10.100.28.138   <none>        8080/TCP   6h59m
+```
+
+Any pod from your cluster can now access this endpoint:
+
+```console
+curl 10.100.28.138:8080
+<?xml version="1.0" encoding="UTF-8"?><ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><ID>anonymous</ID><DisplayName></DisplayName></Owner><Buckets></Buckets></ListAllMyBucketsResult>
+```
+
+It is also possible to use the internally registered DNS name:
+
+```console
+curl rook-ceph-rgw-my-store.rook-ceph:8080
+<?xml version="1.0" encoding="UTF-8"?><ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><ID>anonymous</ID><DisplayName></DisplayName></Owner><Buckets></Buckets></ListAllMyBucketsResult>
+```
+
+The DNS name is created  with the following schema `rook-ceph-rgw-$STORE_NAME.$NAMESPACE`.
```

Comment:
This applies for the converged mode as well , how about having  common section in which `checking the rgw end point`

Reply:
I think this is covered already by the "Consume the Object Storage" section right?